ksiren/src/main/java/com/brightspace/ksiren/Action.kt

```diff
@@ -15,14 +15,35 @@ package com.brightspace.ksiren
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+interface ActionBase : JsonSerializable {
+	val name: String
+	val classes: List<String>
+	val method: String
+	val href: String
+	val title: String?
+	val type: ContentType
+	val fields: List<FieldBase>
+
+	fun hasField(name: String): Boolean = this.fields.any() { it.name == name }
```

Comment:
Nitpick: use a named parameter instead of it, and include the function in the ()s for clarity purposes.
`fun hasField(name: String): Boolean = this.fields.any( {field -> field.name == name } )`

Reply:
Good to know. I was basing patterns on Kotlin docs (https://kotlinlang.org/docs/reference/collection-transformations.html) and some other uses (eg. in `JsonUtils`) thinking `it` was the idiomatic "item of this collection" in simple cases, so thanks for the correction!

Reply:
Hm, I get `Lambda argument should be moved out of parentheses` using that syntax, which is something I noticed with a lot of pre-existing code as well. I was following [these docs](https://kotlinlang.org/docs/reference/coding-conventions.html#lambda-formatting), but if this is better written as a block or some other format, happy to update it.

Reply:
Hmm, I guess maybe it's industry practice to separate them out then?  I'm okay with it staying out if the linter is complaining.